brave/brave-core
# Reorganize resources in rewards folder

Resolves brave/brave-browser#4003

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- make sure that rewards page loads
- make sure that tip banner loads
- make sure that rewards internals loads

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
